v-next/hardhat-errors/src/descriptors.ts

```diff
@@ -145,6 +145,8 @@ Please double check whether you have multiple versions of the same plugin instal
       websiteTitle: "Hardhat project already created",
       websiteDescription: `Cannot create a new Hardhat project, the current folder is already associated with a project.`,
     },
+    // TODO: This error is no longer relevant, is it OK to remove it?
+    // Should we update the following error numbers afterwards?
     NOT_INSIDE_PROJECT_ON_WINDOWS: {
```

Comment:
Should/could we remove this error now?

Reply:
yes, this can be removed now

Reply:
When you remove them, the tests will force you to update the error numbers